firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarAction.swift

```diff
@@ -70,19 +70,22 @@ class ToolbarMiddlewareAction: Action {
     }
 }
 
-class ToolbarMiddlewareUrlChangeAction: ToolbarMiddlewareAction {
+final class ToolbarMiddlewareUrlChangeAction: ToolbarMiddlewareAction {
     let url: URL?
+    let lockIconImageName: String?
```

Comment:
Why is the image name optional? From what I can see `ToolbarMiddlewareUrlChangeAction` is only used once and in that case we have an image.